aws/cloudformation/components/database.yml.erb

```diff
@@ -280,7 +328,7 @@
     Type: AWS::RDS::DBCluster
     Properties:
       DBClusterIdentifier: !Sub "${AWS::StackName}-cluster"
-      DBClusterParameterGroupName: !Ref AuroraClusterDBParameters
+      DBClusterParameterGroupName: !Ref Aurora2ClusterDBParameters
```

Comment:
[update requires some interruptions](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbinstance.html#cfn-rds-dbinstance-dbparametergroupname)

Is that okay/expected?

Reply:
oh, thanks for catching that. We’re hopefully not actually changing the Physical resource that’s being assigned to the cluster and instance (which would happen if we changed the [DBParameterGroupName](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbparametergroup.html#cfn-rds-dbparametergroup-dbparametergroupname) attribute, but it’s not worth the risk. I’ll leave the existing ParameterGroup name unchanged. Having an `Aurora3…` name for the new Resources might be a little confusing, but will totally make sense when we upgrade to Aurora Engine 4!

Reply:
Note in the Database Instance Event log when it applied the fix for this issue (keeping the existing ParameterGroup Logical ID and Physical ID) that each instance in the cluster restarted, which is NOT good. Glad you caught this.

<img width="682" alt="image" src="https://github.com/code-dot-org/code-dot-org/assets/2157034/84aa4730-1048-44d5-866c-395b5037c716">